.gitmodules

```diff
@@ -99,3 +99,6 @@
 	path = third-party/lwext4/lwext4
 	url = ../lwext4.git
 	branch = mudita
+[submodule "third-party/dr_libs"]
+	path = third-party/dr_libs
+	url = git@github.com:mudita/dr_libs.git
```

Comment:
please change the URL to the HTTPS instead of SSH

Reply:
Fixed